### DISCUSSION TITLE
Available variables for the compose file?

### DISCUSSION THREAD

**Discussion by Pshemas** [semaphoreui/semaphore#2033] [Q&A] [answered]
It's probably just me not finding things... but is there part of the doc desciribing all the environment variables you can use in docker compose file?

I'd like to set as much possible there, so I'd only do as few edits to config.json as possible :) .

**Comment by tboerger** [comment, marked as answer]
Not a documentation for that, but at least you can see all available environment variables at https://github.com/semaphoreui/semaphore/blob/09cf43e1262e8f5d401a2bbc29cea8b860225a76/util/config.go and https://github.com/semaphoreui/semaphore/blob/09cf43e1262e8f5d401a2bbc29cea8b860225a76/deployment/docker/server/server-wrapper
